ministryofjustice/correspondence_tool_staff
# CT-3481 Make count tab values to reflect filters

## Description
 Makes the tabs on the "my open cases page" reflect the totals of the cases returned from the filters.

## Self-review checklist



### Screenshots

### Related JIRA tickets
[CT-3481](https://dsdmoj.atlassian.net/browse/CT-3481)


### Manual testing instructions
- create some complaint cases and assign them to yourself, make sure they are of different types.
- go to cases > my open cases 
- use the filters
- the numbers in the tabs should now reflect the totals of the results.
